content/soups and stews/Middle-Eastern Chili.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Middle-Eastern Chili
+category: soups and stews
+date: 2025-08-21
+recipeYear: "2025"
+recipeDecade: 2020s
+author: Liza Desyatkova
+prepTime: 15m
+cookTime: 30m
+vegetarian: true
+---
+
+### Ingredients
+
+#### Part 1
```

Comment:
Part headings are not required if the instructions are not split into multiple parts